src/components/books/AddBook.js

```diff
@@ -1,21 +1,45 @@
-// import React from "react";
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../../redux/books/books';
 
-const AddBook = () => (
-  <article>
-    <h3>ADD NEW BOOK</h3>
-    <form>
-      <input placeholder="Book title" />
-      <select placeholder="Categorie">
-        <option value="" selected disabled hidden>
-          Categorie
-        </option>
-        <option>Action</option>
-        <option>Science Fiction</option>
-        <option>Economy</option>
-      </select>
-      <button type="submit">ADD BOOK</button>
-    </form>
-  </article>
-);
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+
+  const handleChange = (e) => setTitle(e.target.value);
+
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+    const book = {
+      id: uuidv4(),
+      title,
+    };
+
+    dispatch(addBook(book));
+    setTitle('');
+  };
+
+  return (
+    <article>
+      <h3>ADD NEW BOOK</h3>
+      <form onSubmit={submitBookToStore}>
```

Comment:
- [ ] Here you could add the input for the author, Is important for the project. 🌵

Reply:
Hi, Victor!

Yes, but in the design, given us from Microverse i do not see any author input. Only for the title. That is why I didn't add it.

Reply:
Hi, Victor!

Yes, but in the design, given us from Microverse i do not see any author input. Only for the title. That is why I didn't add it.

Reply:
Could you show me this design given? I'd like to see it. 😄

Reply:
Could you show me this design given? This will help to improve the documentation. 😄